README.md

```diff
@@ -173,7 +161,13 @@ If you are running ArchLinux, run the following command instead:
 echo fs.inotify.max_user_watches=524288 | sudo tee /etc/sysctl.d/40-max-user-watches.conf && sudo sysctl --system
 ```
 
-### WebStorm always chooses the wrong auto import path in a workspace
+## WebStorm always chooses the wrong auto import path in a workspace
 Go to Settings/Preferences dialog (⌘,) -> Code Style -> Editor -> choose JavaScript or TypeScript -> Imports tab -> select **Only in files outside specified paths**
 
 [![Webstorm_settings](https://user-images.githubusercontent.com/2495032/57608810-f5f6fb80-756d-11e9-8403-e33f17c04212.png)](https://github.com/nrwl/nx/issues/83)
+
+# License 
+Except where otherwise specified, source code in this repository is licensed under
+the CDDL license.
```

Comment:
I would remove the license section from the readme as it is already described in the LICENSE.md

We also use MIT not CDDL